Casks/t/tastytrade.rb

```diff
@@ -1,17 +1,29 @@
 cask "tastytrade" do
   arch arm: "-aarch64"
 
-  version "2.5.4"
-  sha256 arm:   "51f21ecd02cd7b0acf13d80f712e87a2a163d79a8b1721cebc78559deb1615fe",
-         intel: "da585b646afba35104957cbfb433c4b27cc9f1ee6fd69f3a045397c3a27d5c7f"
+  version "2.8.0"
+  sha256 arm:   "62532bc7b23a439e501c0eff1c3bab216adcbc5c64c05539d06fcdf5d1705088",
+         intel: "98ace9292f65022780328fb3a61c98c6542e62046390770d16b8942a9b9e6be8"
 
   url "https://download.tastytrade.com/desktop-#{version.major}.x.x/#{version}/tastytrade-#{version}#{arch}.dmg"
   name "tastytrade"
   desc "Desktop trading platform"
   homepage "https://tastytrade.com/technology/"
 
   livecheck do
-    skip "No version information available"
+    url "https://tastytrade.com/page-data/technology/page-data.json"
+    strategy :json do |json|
+      static_hashes = json["staticQueryHashes"]
+
+      version = static_hashes.map do |static_hash|
```

Comment:
```suggestion
      version = json["staticQueryHashes"]&.map do |static_hash|
```

The `staticQueryHashes` value may not exist, so we should guard against that. We can also simply inline `json["staticQueryHashes"]`, since we don't use the `static_hashes` variable anywhere else.